core/tests/performance_tests/base.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common utilities for performance test classes"""
+
+import unittest
+
+from core.tests.performance_framework import perf_services
+
+
+class TestBase(unittest.TestCase):
+    """Base class for performance tests."""
+
+    def setUp(self):
+        self.data_fetcher = perf_services.SeleniumPerformanceDataFetcher(
+            browser='chrome')
+        self.page_metrics = None
+
+    def _set_page_session_stats(self, page_url):
```

Comment:
Maybe these can be named to better describe the intention of the test writer. For instance:

``_record_page_session_stats``

And, similarly, the other methods might be like

``_record_cached_page_session_stats`` or ``_record_page_session_timings``. Small adjustments to make things read a bit better.